data-pipeline/src/data_pipeline/data_types/coverage.py

```diff
@@ -13,7 +13,7 @@ def prepare_coverage(coverage_path: str, filter_intervals: Optional[List[str]] =
 
     # Specific to v4, the data is contained in a coverage_stats field and we take the first element
     if "coverage_stats" in coverage_fields:
-        if type(coverage.coverage_stats) == hl.expr.expressions.typed_expressions.ArrayStructExpression:
+        if type(coverage.coverage_stats) is hl.expr.expressions.typed_expressions.ArrayStructExpression:
```

Comment:
Honestly, this change was just because the linter we use, [`ruff`](https://docs.astral.sh/ruff/linter/) was complaining about these lines and it can be nice to just defer to a linter/formatter for stylistic decisions. 

With the change, they _should_ be equivalent seeing as its checking a type, so the underlying types of the two should be both the same address in memory (what `is`) checks for, and the same value (what `==` checks for)